src/Components/Stats/Stats.js

```diff
@@ -4,12 +4,26 @@ import style from './Stats.module.scss'
 const Stats = () => {
   return (
     <article>
-      <h2 className={style.prompt}>⏳ Stats Page Coming Soon! ⏳</h2>
+      <h2 className={style.prompt}>
+        <span role="img" alt="hourglass emoji" aria-label="hourglass emoji">
```

Comment:
Wow, didn't know this was the standard for emojis - good to know moving forward!